e2e_samples/temperature_events/Functions/TemperatureEventsProj/host.json

```diff
@@ -0,0 +1,11 @@
+{

```

Comment:
@jmostella Just FYI for future development.

When we host multiple functions in Function App (Azure resource), these functions utilize same host.json. It's OK if we use default host.json, but if we want to tweak some parameter for event-hub triggers, we need to consider 1) Prepare multiple Function App and enable only one function in each Function App 2) Overwrite host.json parameter by app settings like my repo.
https://github.com/NT-D/eh-func-iac/blob/main/terraform/function.tf#L49

Reply:
Yes I assumed that we would follow approach that was used in your blog